students/swietczak_monika/lesson_01_basics/car_route.py

```diff
@@ -0,0 +1,7 @@
+# Read an integer:
+import math
+N = int(input())
```

Comment:
Capital letters should be used only for constants e.g.
MY_NAME = "Kamil"

Please use lower case instead:
n = int(input())

Reply:
It will be great to include text description whenever user is prompt for input
n = int(input("How many kilometers per day: "))